packages/core-data/src/entity-provider.js

```diff
@@ -145,14 +145,19 @@ const parsedBlocksCache = new WeakMap();
  * `BlockEditorProvider` and are intended to be used with it,
  * or similar components or hooks.
  *
- * @param {string} kind         The entity kind.
- * @param {string} name         The entity name.
+ * @param {string} kind              The entity kind.
+ * @param {string} name              The entity name.
  * @param {Object} options
- * @param {string} [options.id] An entity ID to use instead of the context-provided one.
+ * @param {string} [options.id]      An entity ID to use instead of the context-provided one.
+ * @param {string} [options.context] The context param to be passed to the REST API.
  *
  * @return {[WPBlock[], Function, Function]} The block array and setters.
  */
-export function useEntityBlockEditor( kind, name, { id: _id } = {} ) {
+export function useEntityBlockEditor(
+	kind,
+	name,
+	{ id: _id, context = 'edit' } = {}
```

Comment:
I don't like the context prop too much here. It's also weird to pass `context: view` to `getEditedEntityRecord`.

Reply:
I actually think this hook is probably not for "previewing" but should only be used for "editing".